app/middleware/auth.js

```diff
@@ -0,0 +1,37 @@
+import 'babel-polyfill';
+import jwt from 'jsonwebtoken';
+
+/**
```

Comment:
This comment is placed wrongly, it is not above the comment for which it is providing commentary for

Reply:
Done